backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/handlingunits/process/ReceiptScheduleBasedProcess.java

```diff
@@ -61,7 +61,7 @@ public abstract class ReceiptScheduleBasedProcess extends JavaProcess implements
 	private final IHUAttributesBL huAttributesBL = Services.get(IHUAttributesBL.class);
 	private final ILotNumberBL lotNumberBL = Services.get(ILotNumberBL.class);
 
-	private Optional<String> lotNumberFromSeq = null;
+	private Optional<String> lotNumberFromSeq = Optional.empty();
```

Comment:
actually we need to revert this, pls check:  `private String getOrLoadLotNoFromSeq(final @NonNull I_M_ReceiptSchedule receiptSchedule)` as if we leave it like it is now, the lotNo will never be loaded

Reply:
https://github.com/metasfresh/metasfresh/pull/15110